carbon-design-system/carbon
# RadioTile types addition Addition issue #13572

Closes #13572

Added TypeScript types to the RadioTile component.

Changelog
New

Added TypeScript types to the RadioTile component.
Changed

Updated the light prop type to match its actual usage.
Testing / Reviewing
To verify this PR:

Clone the repository.
Checkout this branch.
Build the project.
Run tests to ensure no new issues are introduced.
Verify that the light prop type matches the actual usage in the code.
This PR includes updates to the RadioTile component to provide TypeScript types for its props. It also updates the light prop to match its actual usage, improving code quality and maintainability.

Contributors, please review and provide feedback on these changes. Thank you!
